src/pages/Home.js

```diff
@@ -136,35 +125,13 @@ class Home extends Component {
                   src: duosDiagram
                 })
               ]),
-              div({className: 'row', style: { background: 'white', margin: '0' } }, [
-                div({className: 'col-md-5', style: {display: 'flex', justifyContent: 'center'}}, [
-                  button({className: 'btn-primary', style: buttonStyle}, [
-                    a({
-                      href: 'https://broad-duos.zendesk.com/hc/en-us/articles/4404601291163-Registering-a-Dataset-in-DUOS',
-                      style: { color: '#fff' },
-                      target: '_blank'
-                    }, ['Register a dataset in DUOS'])
-                  ])
-                ]),
-                div({className: 'col-md-2', style: {display: 'flex', justifyContent: 'center'}}, []),
-                div({className: 'col-md-5', style: {display: 'flex', justifyContent: 'center'}}, [
-                  isLogged ?
-                    button({className: 'btn-primary', style: buttonStyle}, [
-                      a({
-                        href: '/dataset_catalog',
-                        style: { color: '#fff' }
-                      }, ['Submit a Data Access Request'])
-                    ]) :
-                    h(SignIn, { props: this.props, onSignIn, history, customStyle: buttonStyle})
-                ])
-              ]),
             ])
           ]),
           div({className: 'row', style: {background: '#eff0f2', margin: '50px 0', padding: '48px 0 60px 0'}}, [
             div({className: 'col-lg-4 col-md-4'}, [
-              p({style: header}, ['Are you a DAC member?']),
```

Comment:
The failing test is looking for this text on the page - it could be just as simple as changing the test to search for the text `DUOS for DACs` instead.

Reply:
In the file `cypress/e2e/home.cy.js`, lines 10-12 are no longer true based on the changes to the home page that you made. Just delete those lines and the tests should start passing again.